static/data/meetings/notes/2025-05-06/index.md

```diff
@@ -0,0 +1,305 @@
+     
+Alex thinks they may have a different use case.
+     
+Paul is concerned if we do just the storage, image, common in one repo, it won't help as much without Buildah and Podman also involved, as far as catching issues in changes that cause problems in Buildah or Podman.
+     
+Brent is concerned about doing this now.  He's not sure it's the right time now, given the ongoing CNCF move.  Also, he's not sure the benefit will outweigh the pain cured.
+    
+Paul is not sure this makes things less complicated.  Especially considering when an image needs to be updated.  CI wise, he thinks it will be beneficial.  The complexity added may be too much.
+     
+This will be handy when changes need to be made in Storage and Image at the same time.  It would be handy for vendoring into Podman, less special vendoring.
+     
+Matt is very in favor of this.  He thinks one win is getting more eyes on repositories that don't get as much attention.  
+     
+Gerry, having a monorepo would be beneficial.  He would probably learn more about the Storage, Image, and Common repos.
+     
+Giuseppe also thinks it would be beneficial.  He wouldn't have to do WIP test PRs for image and common if he changed storage.
+     
+Tom would like it to, and would like to see a POC sooner.
+     
+     
+#### Open discussion - (50:55 in the video) -
+1. None
+
+### Next Cabal Meeting: Tuesday, July 1, 2025, 11:00 a.m. EDT (UTC-4)
+
+#### Possible Topics
+1. None
+
+### Next Community Meeting: Tuesday, June 3, 2025, 11:00 a.m. EDT (UTC-4)
+
+#### Possible Topics:
+1. None
+
+Meeting finished at 11:54 a.m.
+
+### Raw Meeting Chat:
+
+```
+You
+11:00 AM
+HackMD for notes: https://hackmd.io/gQCfskDuRLm7iOsWgH2yrg?both
+keep
+Pinned
+You
+11:15 AM
+Any issues for projects that currently vendor in say c/image?
+Miloslav Trmac
+11:19 AM
+I’m not sure a smart build system can make a difference - if a c/storage PR touches store.go or drivers/overlay, it is _technically correct_ to test ~the whole stack on top
+Alex Guidi
+11:25 AM
+In oc-mirror we were using go workspaces (monorepo) in order to keep the two oc-mirror versions working (v1 and v2), as you said all the modules will use the same dependencies, this is great when modules are going to evolve together, but when you want to bump one dependency in only one module, things could get trick because maybe it would require major changes in another module. Only sharing my oc-mirror experience here, sometimes our oc-mirror v1 was holding the improvements of oc-mirror v2 bec
+ because of the dependencies being shared between them.
+Dan Walsh
+11:51 AM
+Is this just storage, image, and common merged together? Or Buildah and Podman as well?
+Matt Heon
+11:52 AM
+Initially, storage/image/common, see what breaks and how it works, then if it is going well, expand
+Mohan Boddu
+11:53 AM
+Thank you Jan
+```
+
+### Raw Google Meet Transcript
```

Comment:
Do you actually want to publish that? I just took a brief look and there are ton of incorrect things in there. If we publish the recording is there a need to publish this?

Reply:
I'm not clear about what you're objecting to.  The Google Meet transcript is the chat that went on during the meeting and doesn't get captured in the transcript.  We've always posted the meeting notes based on what I heard and then wrote, as well as the full transcript, since we started capturing it several months or more ago.

If there are inaccuracies in what I recorded in the notes, I'm happy to change them.

Reply:
I am not talking about the notes you took. I am talking about this automatic transcript below. It is full of errors and I have a really hard time even making sense of a lot things it writes so I (personally) don't consider in a  state that should be published. Like there are a ton of sentences I just cannot make sense of and things like "portman" instead of podman.
 
If you have been doing this for months I wouldn't know because I don't review these PRs normally and just looked at it because I looked at another podman.io PR.

Anyway I won't block over that but I don't understand the value that this is adding.

Reply:
FWIW, we've been adding the "raw" notes for several months now, if not over a year.  Despite the bad note-taking they almost always have, there are some people who will reference them against the video to see what was said.  I know I have personally to verify a particular comment that I thought I heard while cleaning up the notes.

Reply:
Feel free to merge as usual then